src/parser/parseMeta.js

```diff
@@ -102,6 +106,9 @@ var parseMeta = (html, url) => {
     if (authorAttrs.includes(property) || authorAttrs.includes(name)) {
       entry.author = content;
     }
+    if (publishedTimeAttrs.includes(property) || publishedTimeAttrs.includes(name)) {
+      entry.publishedTime = content;
```

Comment:
We could use `new Date(content)` here but since it's a non-standard meta I doubt it would parse correctly for every article